data/sampletracking_multiome.csv

```diff
@@ -1,3 +1,3 @@
-date,run_pipeline,Channel Name,sampleid,method,sub_method,condition,replicate,tissue,Lane,Index,project,project_orig,reference,chemistry,flowcell,seq_dir,min_umis,min_genes,percent_mito,cellbender_expected_cells,cellbender_total_droplets_included,focus,append,calc_signature_scores,introns
-2022_08_08,TRUE,channel1,SCRSP-11107,multiome,rna,nuclei,channel1,pbmc,*,SI-TT-G7,FINNGEN,FINNGEN,GRCh38-2020-A,SC3Pv3,H3VFYDRX2,gs://fc-secure-d4adbbf9-8265-4a5c-b14f-23a5f1b5c4f9/bcls/220602_SL-NSA_0375_AH3VFYDRX2/,400,200,20,1000,20000,,,,TRUE
-2022_08_08,TRUE,channel1,SCRSP-11107,multiome,atac,nuclei,channel1,pbmc,*,SI-NA-B4,FINNGEN,FINNGEN,GRCh38-2020-A_arc_v2.0.0,SC3Pv3,HYG2JDRXY,gs://fc-secure-d4adbbf9-8265-4a5c-b14f-23a5f1b5c4f9/bcls/220602_SL-NSA_0374_BHYG2JDRXY/,400,200,20,1000,20000,,,,TRUE
\ No newline at end of file
+date,run_pipeline,Channel Name,sampleid,method,sub_method,condition,replicate,tissue,Lane,Index,Index2,Index3,Index4,instrument_platform,instrument_type,read1_cycles,read2_cycles,index1_cycles,index2_cycles,create_fastq_for_index_reads,trim_umi,override_cycles,project,project_orig,reference,chemistry,flowcell,seq_dir,min_umis,min_genes,percent_mito,cellbender_expected_cells,cellbender_total_droplets_included,focus,append,calc_signature_scores,introns

```

Comment:
I think it might be better to just keep the channel index SI-TT/NA-XX on the sampletracker and extract the corresponding indices (be it i5 and i7 for GEX and index1-4 for ATAC) in the backend using the reference sheet. I think filling out these columns one by one doesn't scale very well.

Reply:
I think override_cycles can be inferred from read1_cycles,read2_cycles,index1_cycles,index2_cycles + sub_method